ember-element-helper/package.json

```diff
@@ -54,8 +54,8 @@
     "prepublishOnly": "pnpm build"
   },
   "dependencies": {
-    "@embroider/addon-shim": "^1.8.3",
-    "@embroider/util": "^1.0.0"
+    "@embroider/addon-shim": "^1.8.7",
```

Comment:
in particular, addon-shim did not have a dependency declared on broccoli-funnel, but because broccoli-funnel is required/imported, it must be declared.